pkg/handlers/auth/user.go

```diff
@@ -66,3 +66,44 @@ func CreateUpdate(c *gin.Context) {
 // @Failure 500 {object} meta.HTTPResponseError
 // @Router /api/user [put]
 func UpdateForSDK() {}
+
+// @Id ListUsers
+// @Summary List user
+// @Tags    User
+// @Produce json
+// @Success 200 {object} []meta.User
+// @Failure 500 {object} meta.HTTPResponseError
+// @Router /api/user [get]
+func ListUser(c *gin.Context) {
+	users, err := auth.GetUsers()
+	if err != nil {
+		c.JSON(http.StatusInternalServerError, meta.HTTPResponseError{Error: err.Error()})
+		return
+	}
+
+	for _, u := range users {
```

Comment:
Instead of making the fields blank I would map the `User` domain object to a adapting HTTP model without the fields.

Reply:
I just merge in single file. [old file](https://github.com/zinclabs/zinc/blob/main/pkg/handlers/auth/list.go#L42)